registry/storage/driver/middleware/cloudfront/middleware_test.go

```diff
@@ -50,6 +50,6 @@ pZeMRablbPQdp8/1NyIwimq1VlG0ohQ4P6qhW7E09ZMC
 		t.Fatal(err)
 	}
 	if storageDriver == nil {
-		t.Fatal("Driver couldnt be initialized.")
+		t.Fatal("Driver couldn't be initialized.")
```

Comment:
nititest of nits (take it or leave it): I know it's testing code but I don't really like contractions in errors/logging, I know this isn't your change but you're already here so why not, also let's kill this period:

```suggestion
		t.Fatal("Driver could not be initialized")
```